plenoptic/simulate/models/naive.py

```diff
@@ -28,4 +38,180 @@ def forward(self, img):
             a clone of the input image
 
         """
-        return img.clone()
+        y = 1 * img
+        return y
+
+
+class Linear(nn.Module):
+    r"""Simplistic linear convolutional model:
+    It splits the input greyscale image into low and high frequencies.
+
+    Parameters
+    ----------
+    kernel_size:
+        Convolutional kernel size.
+    pad_mode:
+        Mode with which to pad image using `nn.functional.pad()`.
+    default_filters:
+        Initialize the first three filters to: low-pass and high-pass on/off.
```

Comment:
this description is wrong (I think it describes the old LinearNonlinear?).

Reply:
Fixed.